lib/tlIO/FFmpegReadVideo.cpp

```diff
@@ -539,12 +549,16 @@ namespace tl
             {
                 avcodec_flush_buffers(_avCodecContext[_avStream]);
 
+                AVRational speed = _avFormatContext->streams[_avStream]->avg_frame_rate;
```

Comment:
I think it would be better to make ```speed``` a member variable so that the logic is only needed once when opening the file.

Reply:
Modified in new commit.